### PR TITLE
Docs: Improved PMREMGenerator page.

### PR DESCRIPTION
**Pull request by mrdoob**
Related issue: https://github.com/mrdoob/three.js/pull/23428#discussion_r804947975

**Description**

@elalish @WestLangley How does the explanation sound to you?

### CODE REVIEW

File: docs/api/en/extras/PMREMGenerator.html [hunk 1]
```diff
@@ -15,7 +15,36 @@ <h1>[name]</h1>
 			CubeUV format that allows us to perform custom interpolation so that we can support nonlinear formats such as RGBE.
 			Unlike a traditional mipmap chain, it only goes down to the LOD_MIN level (above), and then creates extra even more
 			filtered 'mips' at the same LOD_MIN resolution, associated with higher roughness levels. In this way we maintain
-			resolution to smoothly interpolate diffuse lighting while limiting sampling computation.
+			resolution to smoothly interpolate diffuse lighting while limiting sampling computation.<br/><br/>
+
+			Note: The minimum [page:MeshStandardMaterial]'s roughness depends on the size of the provided texture:
```
Comment (elalish) [inline docs/api/en/extras/PMREMGenerator.html:20]: You might also note this is a slightly conservative bound. If your render has small dimensions or the shiny parts have a lot of curvature, you may still be able to get away with a smaller texture size.

File: docs/api/en/extras/PMREMGenerator.html [hunk 2]
```diff
@@ -15,7 +15,36 @@ <h1>[name]</h1>
 			CubeUV format that allows us to perform custom interpolation so that we can support nonlinear formats such as RGBE.
```
Comment (WestLangley) [inline docs/api/en/extras/PMREMGenerator.html:15]: Does the RGBE comment still apply?

Comment (elalish) [reply docs/api/en/extras/PMREMGenerator.html:15]: No, good catch, since it's all float now.

Comment (mrdoob) [reply docs/api/en/extras/PMREMGenerator.html:15]: +1

File: docs/api/en/extras/PMREMGenerator.html (hunk 1 above)
Comment (mrdoob) [reply docs/api/en/extras/PMREMGenerator.html]: Added that additional info verbatim.

**Comment by mrdoob** [comment]
Merging for now...

**Pull request merged**